test/test_integration.py

```diff
@@ -115,6 +115,81 @@ def test_Spin2_backend() -> None:
     )
 
 
+def test_integration_global_phase() -> None:
```

Comment:
Can we add a comment somewhere in the code saying where a global phase is expected to be introduced?